lib/Server.js

```diff
@@ -754,7 +755,7 @@ class Server {
     // TODO remove `{}` after drop webpack v4 support
     const compilerWatchOptions = compilerOptions.watchOptions || {};
     /**
-     * @param {WatchOptions & WebpackConfiguration["watchOptions"]} watchOptions
+     * @param {WatchOptions & { aggregateTimeout?: number, ignored?: AnymatchMatcher | string | RegExp | string[], poll?: number | boolean }} watchOptions
```

Comment:
Ideally we should extend type here, i.e. `WebpackConfiguration["watchOptions"] & { ignored?: AnymatchMatcher | string | RegExp | string[] }`

Reply:
I did that at first, but it was throwing lint errors with this.